src/mock/point.js

```diff
@@ -0,0 +1,26 @@
+import { getRandomInteger } from '../util';
+import dayjs from 'dayjs';
+
+const generateDate = () => {
+  const maxDaysGap = 2000;
+  const daysGap = getRandomInteger(-maxDaysGap, maxDaysGap);
+
+  return dayjs().add(daysGap, 'm').toDate();
+};
+
+export const generatePoint = () => {
+  const dateFrom = generateDate();
+  let dateTo = generateDate();
+  while (dateTo <= dateFrom) {
+    dateTo = generateDate();
+  }
+  return {
+    'basePrice': 222,
```

Comment:
В объекте ключ всегда строка, нет необходимости оборачивать дополнительно
```suggestion
    basePrice: 222,
```